app/tests/integration/nimbus/test_e2e_create_experiment.py

```diff
@@ -103,14 +110,14 @@ def test_create_new_experiment_remote_settings(selenium, base_url):
             item.click()
             break
     record = kinto_dashbard.record
-    record.approve()
+    record.action()
     selenium.get(base_url)
     # refresh until the experiment shows up
     for attempt in range(45):
         try:
             home = HomePage(selenium, base_url).wait_for_page_to_load()
             new_experiments = len(home.tables[0].experiments)
-            assert new_experiments != current_experiments
+            assert new_experiments > current_experiments
```

Comment:
nit: Do we want to add some assertion message to this? (similar to `assert summary_page.rejected_text, "Rejected text box did not load"` below). Or maybe unneeded since we're catching the exception anyways below